participant-manager/src/assets/css/root.scss

```diff
@@ -1,4 +1,5 @@
 @import "./colors.scss";
```

Comment:
Should this have copyright header?

Reply:
Hi @jkrogers18,  we include copyright headers in only SVG images, not in HTML and CSS files.